vms/platformvm/state/state.go

```diff
@@ -831,6 +833,64 @@ func (s *state) DeleteExpiry(entry ExpiryEntry) {
 	s.expiryDiff.DeleteExpiry(entry)
 }
 
+func (s *state) GetCurrentValidatorSet(ctx context.Context, subnetID ids.ID) (map[ids.ID]*validators.GetCurrentValidatorOutput, uint64, error) {
+	result := make(map[ids.ID]*validators.GetCurrentValidatorOutput)
```

Comment:
would it make sense to check `currentStakers` is not nil?

Reply:
currentStakers are initialized to be empty in `New`